<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SimHitControllerForm</class>
 <widget class="QWidget" name="SimHitControllerForm">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>454</width>
    <height>310</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Sim Hits</string>
  </property>
  <layout class="QHBoxLayout">
   <property name="spacing">
    <number>6</number>
   </property>
   <property name="margin">
    <number>9</number>
   </property>
   <item>
    <layout class="QVBoxLayout">
     <property name="spacing">
      <number>6</number>
     </property>
     <property name="margin">
      <number>0</number>
     </property>
     <item>
      <layout class="QVBoxLayout">
       <property name="spacing">
        <number>6</number>
       </property>
       <property name="margin">
        <number>0</number>
       </property>
       <item>
        <layout class="QHBoxLayout">
         <property name="spacing">
          <number>6</number>
         </property>
         <property name="margin">
          <number>0</number>
         </property>
         <item>
          <widget class="QLabel" name="lblGeneral">
           <property name="text">
            <string>Select hits to show</string>
           </property>
          </widget>
         </item>
         <item>
          <spacer>
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>71</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout">
         <property name="spacing">
          <number>6</number>
         </property>
         <property name="margin">
          <number>0</number>
         </property>
         <item>
          <layout class="QVBoxLayout">
           <property name="spacing">
            <number>6</number>
           </property>
           <property name="margin">
            <number>0</number>
           </property>
          </layout>
         </item>
        <item>
        <widget class="QGroupBox" name="groupBox_Nu">
        <property name="title">
            <string>Neutrino</string>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout_3">
            <item>
            <widget class="QCheckBox" name="chbxEmulsionHits">
            <property name="text">
            <string>Emulsion Hits</string>
            </property>
            </widget>
            </item>
        </layout>
        </widget>
        </item>         
         <item>
          <widget class="QGroupBox" name="groupBox">
           <property name="title">
            <string>Scintillator</string>
           </property>
           <layout class="QVBoxLayout" name="verticalLayout">
            <item>
             <widget class="QCheckBox" name="chbxVetoNuHits">
              <property name="text">
               <string>VetoNu Hits</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QCheckBox" name="chbxVetoHits">
              <property name="text">
               <string>Veto Hits</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QCheckBox" name="chbxTriggerHits">
              <property name="text">
               <string>Trigger Hits</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QCheckBox" name="chbxPreshowerHits">
              <property name="text">
               <string>Preshower Hits</string>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
         </item>


         <item>
          <layout class="QVBoxLayout">
           <property name="spacing">
            <number>6</number>
           </property>
           <property name="margin">
            <number>0</number>
           </property>
          </layout>
         </item>
        </layout>
       </item>
      </layout>
     </item>
         <item>
          <spacer>
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <widget class="QGroupBox" name="groupBox_2">
           <property name="title">
            <string>Tracker</string>
           </property>
           <layout class="QVBoxLayout" name="verticalLayout_2">
            <item>
             <widget class="QCheckBox" name="chbxTrackerHits">
              <property name="text">
               <string>Tracker Hits</string>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
         </item>

     <item>
      <widget class="QGroupBox" name="groupBox_3">
       <property name="title">
        <string>Calorimeter</string>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout_31">
        <item>
         <widget class="QCheckBox" name="chbxCaloHits">
          <property name="text">
           <string>Calorimeter Hits</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <spacer>
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>111</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item>
    <spacer>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>111</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
